.travis.yml

```diff
@@ -87,13 +87,28 @@ matrix:
    - language: rust
      rust: stable
      os: windows
+     env:
+       TARGET=x86_64-pc-windows-msvc
      before_install:
       - choco install golang nasm
       # Update $PATH
       - export PATH="$(powershell -Command '("Process", "Machine" | % { [Environment]::GetEnvironmentVariable("PATH", $_) -Split ";" -Replace "\\$", "" } | Select -Unique | % { cygpath $_ }) -Join ":"')"
      script:
-      - RUSTFLAGS="-D warnings" cargo build --release --verbose
-      - RUSTFLAGS="-D warnings" cargo test --verbose
+      - RUSTFLAGS="-D warnings" cargo build --release --verbose --target=$TARGET
+      - RUSTFLAGS="-D warnings" cargo test --verbose --target=$TARGET
+   - language: rust
```

Comment:
This looks fine, though we might want to consider merging some of these builds together in the future, as each build now takes 30 minutes, partly due to the fact that we have a 5 (?) concurrent workers limit, so a bunch of builds just get stuck waiting for others to complete.

Reply:
Created #295 for optimizing current CI config.